.expeditor/config.yml

```diff
@@ -0,0 +1,27 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
+---
+pipelines:
+  - verify:
+    description: Run pull request verification tests (spec/lint)
```

Comment:
need an indentation. Also, you should also add `public: true` since this is an open source repo.